README.md

```diff
@@ -168,31 +168,31 @@ Ghostfolio is available for various home server systems, including [CasaOS](http
 
 #### Debug
 
-Run `yarn watch:server` and click _Debug API_ in [Visual Studio Code](https://code.visualstudio.com)
+Run `npm run watch:server` and click _Debug API_ in [Visual Studio Code](https://code.visualstudio.com)
 
 #### Serve
 
-Run `yarn start:server`
+Run `npm start:server`
```

Comment:
`npm run start:server`❓

Reply:
There is a separate script to start server,client and storybook for local development. Am I missing something?. I ran it this way as well.

Reply:
I have added **run** 🙂